content/02.manuscript.md

```diff
@@ -1,12 +1,54 @@
+## Introduction
 
-## Title 1
+- Importance of computation in biological research and scope.
+- Importance to reproducibility in computational biologt.
+- Mentioning previous literature and some of their major takeaways and topics not fully covered in here (what kind of languages to learn, advanced text editors/IDEs, 
+- The computational biology best-practices continuum: reproducibility at the personal level, sharing our research with others and over time.
+- Addressing the audience: computational biologists researchers – data analysts, workflow designers, software developers, mathematical modelers, etc.
+- Structure of this manuscript: mention Figure 1 and Figure 2.
+  - **Figure 1:** Funnel structure for better computational biology.
+  - **Figure 2:** Types of computational biology projects per level.
 
-Some text.
+## Level 1: Individual Research
 
-## Title 2
+Goal: How should you manage your computational biology project?
 
-More great stuff.
+Topics:
+- Literate programming: R Markdown, Jupyter Notebooks
+- Version control: Git / GitHub (commits)
+- Software versioning and environment managers (Conda, python-env)
+- Modularize, snippets
+- Coding style: variable naming, linter, pep8, commenting
+- Programming practices: paradigms (object-oriented, procedural), assertions, pair-programming
 
-### Subtitle
+## Level 2: Collaboration
 
-Details
+Goal: How to allow your collaborators to reproduce and interact with your research/software?
+
+Topics:
+- Sharing notebooks: R Markdowns and Jupyter Notebooks (Binder, Google CoLab)
+- Sharing apps: Shiny apps, Dashboard.
+- GitHub (branching, pull requests)
+- Workflow automation: Snakemake (NextFlow, Make, Bash script)
+- Sharing data and metadata
+
+## Level 3: Community
+
+Goal: How to develop and mantain a computational biology project with community feedback over time?
+
+Topics:
+- GitHub releases and semantic versioning
+- Git Flow, GitHub Issues
+- Continous integration and unit tests
```

Comment:
I feel like unit tests is something useful in a project from the beginning, but for sure already at the "Collaboration" level... maybe move up? CI can stay here.